py/client/pydeephaven/session.py

```diff
@@ -418,6 +451,41 @@ def open_table(self, name: str) -> Table:
                 faketable.ticket = None
                 faketable.schema = None
 
+    def publish_table(self, table: Table, shared_ticket: SharedTicket) -> None:
+        """Publishes a table with the given shared ticket for sharing with other sessions.
+
+        Args:
+            table (Table): a Table object
+            shared_ticket (SharedTicket): a SharedTicket object
+
+        Raises:
+            DHError
+        """
+        return self._session_service.publish(table.ticket, shared_ticket.api_ticket)
+
+    def fetch_table(self, ticket: SharedTicket) -> Table:
```

Comment:
We have different operations for retrieving a table by ticket type? That seems weird.

Reply:
We have intentionally decided to not expose the details of tickets and ticket types in the Python APIs in favor of simplicity. I think it has worked well since we only have one higher level object type Table. The introduction of SharedTicket changed that but IMO not enough to remodel the ticket handling for now.